src/PowerShell/Microsoft.WinGet.Client.Engine/Commands/InstallerPackageCommand.cs

```diff
@@ -94,6 +94,8 @@ public sealed class InstallerPackageCommand : InstallCommand
             string psPackageFieldMatchOption,
             string psPackageInstallMode)
         {
+            CatalogPackage? catalogPackage = null;
+
             var result = this.Execute(
                 async () => await this.GetPackageAndExecuteAsync(

```

Comment:
Instead of this, you can modify the return of GetPackageAndExecuteAsync to be a tuple with TResult and the catalog package.